### ISSUE TITLE
Browser crashes on sign in with passkey

### ISSUE THREAD

**Issue by Daleth50** [zen-browser/desktop#1946]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

I tried to sign in into github using passkey from bitwarden extension but zen keeps crashing every time.
On firefox 132.0b2 (64-bit) this does not happen.

Zen version 1.0.1-a.7 (64-bit) (flatpak)

SO: Arch linux with wayland
Kernel: Linux 6.11.1-arch1-1
AMD Ryzen™ 5 3400G
NVIDIA GeForce RTX™ 3050

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.7

### What platform are you seeing the problem on?

Linux

### Relevant log output

```shell
Crash Annotation GraphicsCriticalError: |[0][GFX1]: Error in eglSetDamageRegion: 0x3009 (t=1.3945) |[1][GFX1]: Error in eglSetDamageRegion: 0x3009 (t=1.4045) [GFX1]: Error in eglSetDamageRegion: 0x3009

Crash Annotation GraphicsCriticalError: |[C0][GFX1-]: CompositorBridgeChild receives IPC close with reason=AbnormalShutdown (t=12.6597) Exiting due to channel error.
```

**Comment by RayZ3R0** [comment]
Cannot reproduce this as I use passkeys with bitwarden a lot. Does this still happen on latest version? (a.14 as of this comment)

**Comment by joyal-jij0** [comment]
Yes I am also facing this issue. My version is 1.0.1-a.16 (Firefox 132.0) (aarch64)

**Comment by Daleth50** [comment]
the issue seems to be solved in version 1.0.1-a.16 Flatpak

**Comment by RayZ3R0** [comment]
Alright, will be closing this then
